src/section/AlertSection.js

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import { makeStyles } from "@material-ui/core/styles"
+import InfoIcon from "@material-ui/icons/Info"
+
+import theme from "../theme"
+import palette from "../theme/palette"
+import AlertCo from "../components/alertCo"
+
+const useStyles = makeStyles(() => ({
+  root: {
+    width: "100%",
+    "& > * + *": {
+      marginTop: theme.spacing(2),
+    },
+  },
+}))
+
+const AlertSection = () => {
+  const classes = useStyles()
+  const sampleText =
+    "¡Lo sentimos! No pudimos encontrar tu reserva con la información proporcionada. Por favor, revisa tus datos y vuelve a intentarlo."
+
+  return (
+    <div className={classes.root}>
+      <AlertCo
+        customColor={palette.primary.main}
```

Comment:
Why do you prefer this rather than the color names like "primary" and "secondary"? 🤔

Reply:
I've used this based in another palette implemented in another code.

Reply:
besides, is to show a custom color focus completely in the example shown by Santi in the Copa website